insights/client/constants.py

```diff
@@ -87,3 +87,4 @@ class InsightsConstants(object):
     rhsm_facts_file = os.path.join(os.sep, 'etc', 'rhsm', 'facts', 'insights-client.facts')
     # In MB
     archive_filesize_max = 100
+    insights_archive_tmp_path = '/var/tmp/insights-client'
```

Comment:
```suggestion
    insights_archive_tmp_path = os.path.join(os.sep, 'var', 'tmp', 'insights-client')
```

for consistency.

What about naming this more general like _insights_tmp_path_ and using it for _egg_release_file_ too?